github/src/main/kotlin/gropius/sync/github/NodeSourcerer.kt

```diff
@@ -165,7 +165,7 @@ class NodeSourcerer(
      * @param info The GraphQL data for this label
      * @return a gropius label
      */
-    suspend fun ensureLabel(info: LabelData): Label {
+    suspend fun ensureLabel(imsProjectConfig: IMSProjectConfig, info: LabelData): Label {
```

Comment:
param doc for `imsProjectConfig`

Reply:
Already modified in future PR, will not risk Merge Conflicts